lib/src/transport.dart

```diff
@@ -141,12 +159,12 @@ class Transport implements GeneratedMessageSender {
       request,
       true,
     );
-    final List<int> data = _commandEncoder.convert(command);
-    _socket!.add(data);
+    final data = _commandEncoder.convert(command);
+    _socket!.sendData(data);
   }
 
   Future? close() {
-    return _socket?.close();
+    return _socket?.sink.close();
```

Comment:
Previously _socket was a dart:io web socket, not it is an abstract web socket channel. Maybe it would be better to name it "_channel" or "_socketChanell"? So this is just  a consequence of using the officially recommended web socket abstraction library now.

Reply:
Got it, thx. I think no real need to rename.